src/Application.vala

```diff
@@ -446,15 +445,8 @@ namespace Switchboard {
 
         // Switches to the given plug
         private void switch_to_plug (Switchboard.Plug plug) {
-            if (should_animate_next_transition == false) {
-                stack.set_transition_type (Gtk.StackTransitionType.NONE);
-                should_animate_next_transition = true;
-            } else if (stack.transition_type == Gtk.StackTransitionType.NONE) {
-                stack.set_transition_type (Gtk.StackTransitionType.SLIDE_LEFT_RIGHT);
-            }
-
```

Comment:
Removing this means i.e. when you open Network settings from the indicator, that you see the home screen briefly first before it switches to the plug, which feels janky. Is that something you plan to clean up in a follow-up?

Reply:
Ah hm. Gonna have to dig into this a bit and see if there's a way to temporarily disable animations for Hdy.Deck